homeassistant/components/camera/arlo.py

```diff
@@ -90,3 +93,22 @@ def model(self):
     def brand(self):
         """Camera brand."""
         return DEFAULT_BRAND
+
+    @property
+    def status(self):
+        """Camera Status."""
+        return self._status
+
+    @asyncio.coroutine
+    def async_arm(self):
+        """Camera arm."""
+        self._base_stn.mode = "armed"
```

Comment:
Is there any example as to how to handle I/O, if we cannot do it inside the async methods ?